lib/SILOptimizer/PassManager/PassPipeline.cpp

```diff
@@ -754,6 +754,10 @@ SILPassPipelinePlan::getLoweringPassPipeline(const SILOptions &Options) {
   return P;
 }
 
+static llvm::cl::opt<bool>
+    EnableStackSizeLimitTransform("swift-irgen-enable-stack-size-limit",
+                                  llvm::cl::init(false), llvm::cl::Hidden);
```

Comment:
Why do you want to enable this change with a feature flag? It doesn't look so risky that it cannot be enabled by default with this PR.

Reply:
I put this here since I wanted to get up a PR quickly that was clearly a draft and that could be used to generate a toolchain that @Catfish-Man could play with. So I wanted them to be able to turn it on/off for experimentation or if something came up. I don't think it will be in the final version.